packages/communication-react/review/beta/communication-react.api.md

```diff
@@ -775,6 +775,7 @@ export type CallCompositeOptions = {
     remoteVideoTileMenuOptions?: RemoteVideoTileMenuOptions;
     localVideoTile?: boolean | LocalVideoTileOptions;
     videoTilesOptions?: VideoTilesOptions;
+    notificationOptions?: NotificationStackOptions;
```

Comment:
`Stack` is introduced into the type here but has no context regarding the property name (composite.notificationOptions)

```suggestion
    notificationOptions?: NotificationOptions;
```
or
```suggestion
    notificationOptions?: CompositeNotificationOptions;
```